elections.md

```diff
@@ -65,17 +46,22 @@ representation is achieved. If it is impossible to find sufficient members to
 resign, the entire company’s representation will be removed and new special
 elections held. In the event of a question of company membership (for example
 evaluating independence of corporate subsidiaries) a majority of all
-non-involved Steering Committee members will decide. 
+non-involved Steering Committee members will decide.
+
+### Terms and election cycles
 
-### Initial Election
+Steering committee members are elected to serve two year terms, with roughly
```

Comment:
This explicitly calls out two year terms now

Reply:
I think you do need to spell it out instead of roughly half.  Assuming that you don't want to use numbers because the total seats on the SC might change:

Steering committee members are elected to serve two year terms.  Half of the committee will be re-elected every year, alternating between half rounded up and half rounded down when the committee has an odd number of members (e.g. alternating between 3 and 4 seats out of 7 seats).

Reply:
Also, I think we need to spell out whether we're optimizing for 50% replacement each year, or for SC members having 2-year terms.  Based on what we did with the bootstrap committee, I'm presuming it's the former?